homeassistant/components/schlage/__init__.py

```diff
@@ -0,0 +1,41 @@
+"""The Schlage integration."""
+from __future__ import annotations
+
+from pycognito.exceptions import WarrantException
+import pyschlage
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+from .coordinator import SchlageDataUpdateCoordinator
+
+PLATFORMS: list[Platform] = [Platform.LOCK]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Schlage from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
```suggestion
```

We usually don't set anything in `hass.data` until we get past the point we are sure the integration is likely to set up

Reply:
Done.

Should the scaffold template get updated to reflect this? https://github.com/home-assistant/core/blob/dev/script/scaffold/templates/config_flow/integration/__init__.py#L18